<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>InviteForm</class>
  <widget class="QDialog" name="InviteForm">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>592</width>
        <height>203</height>
      </rect>
    </property>
    <property name="sizePolicy">
      <sizepolicy>
        <hsizetype>5</hsizetype>
        <vsizetype>5</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
    <property name="windowTitle">
      <string>Twinkle - Call</string>
    </property>
    <layout class="QVBoxLayout">
      <item>
        <layout class="QGridLayout">
          <item row="1" column="0">
            <widget class="QLabel" name="inviteTextLabel">
              <property name="text">
                <string>&amp;To:</string>
              </property>
              <property name="buddy" stdset="0">
                <cstring>inviteComboBox</cstring>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="2" column="2">
            <spacer>
              <property name="sizeHint">
                <size>
                  <width>20</width>
                  <height>23</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Vertical</enum>
              </property>
            </spacer>
          </item>
          <item row="2" column="1">
            <widget class="QLineEdit" name="subjectLineEdit">
              <property name="whatsThis" stdset="0">
                <string>Optionally you can provide a subject here. This might be shown to the callee.</string>
              </property>
            </widget>
          </item>
          <item row="1" column="2">
            <widget class="QToolButton" name="addressToolButton">
              <property name="focusPolicy">
                <enum>Qt::TabFocus</enum>
              </property>
              <property name="text">
                <string/>
              </property>
              <property name="shortcut">
                <string>F10</string>
              </property>
              <property name="icon">
                <iconset>kontact_contacts.png</iconset>
              </property>
              <property name="toolTip" stdset="0">
                <string>Address book</string>
              </property>
              <property name="whatsThis" stdset="0">
                <string>Select an address from the address book.</string>
              </property>
            </widget>
          </item>
          <item row="0" column="2">
            <spacer>
              <property name="sizeHint">
                <size>
                  <width>20</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Vertical</enum>
              </property>
            </spacer>
          </item>
          <item row="1" column="1">
            <widget class="QComboBox" name="inviteComboBox">
              <property name="sizePolicy">
                <sizepolicy>
                  <hsizetype>7</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="editable">
                <bool>true</bool>
              </property>
              <property name="maxCount">
                <number>10</number>
              </property>
              <property name="insertPolicy">
                <enum>QComboBox::NoInsert</enum>
              </property>
              <property name="autoCompletion">
                <bool>true</bool>
              </property>
              <property name="whatsThis" stdset="0">
                <string>The address that you want to call. This can be a full SIP address like &lt;b&gt;sip:example@example.com&lt;/b&gt; or just the user part or telephone number of the full address. When you do not specify a full address, then Twinkle will complete the address by using the domain value of your user profile.</string>
              </property>
            </widget>
          </item>
          <item row="0" column="1">
            <widget class="QComboBox" name="fromComboBox">
              <property name="sizePolicy">
                <sizepolicy>
                  <hsizetype>7</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="whatsThis" stdset="0">
                <string>The user that will make the call.</string>
              </property>
            </widget>
          </item>
          <item row="2" column="0">
            <widget class="QLabel" name="subjectTextLabel">
              <property name="text">
                <string>&amp;Subject:</string>
              </property>
              <property name="buddy" stdset="0">
                <cstring>subjectLineEdit</cstring>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="0" column="0">
            <widget class="QLabel" name="fromTextLabel">
              <property name="text">
                <string>&amp;From:</string>
              </property>
              <property name="buddy" stdset="0">
                <cstring>fromComboBox</cstring>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <item>
            <widget class="QCheckBox" name="hideUserCheckBox">
              <property name="text">
                <string>&amp;Hide identity</string>
              </property>
              <property name="shortcut">
                <string>Alt+H</string>
              </property>
              <property name="whatsThis" stdset="0">
                <string>&lt;p&gt;
With this option you request your SIP provider to hide your identity from the called party. This will only hide your identity, e.g. your SIP address, telephone number. It does &lt;b&gt;not&lt;/b&gt; hide your IP address.
&lt;/p&gt;
&lt;p&gt;
&lt;b&gt;Warning:&lt;/b&gt; not all providers support identity hiding.
&lt;/p&gt;</string>
              </property>
            </widget>
          </item>
          <item>
            <spacer>
              <property name="sizeHint">
                <size>
                  <width>181</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
      <item>
        <spacer>
          <property name="sizeHint">
            <size>
              <width>20</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation">
            <enum>Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <item>
            <spacer>
              <property name="sizeHint">
                <size>
                  <width>91</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="okPushButton">
              <property name="text">
                <string>&amp;OK</string>
              </property>
              <property name="default">
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="cancelPushButton">
              <property name="text">
                <string>&amp;Cancel</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <pixmapfunction></pixmapfunction>
  <tabstops>
    <tabstop>inviteComboBox</tabstop>
    <tabstop>subjectLineEdit</tabstop>
    <tabstop>hideUserCheckBox</tabstop>
    <tabstop>addressToolButton</tabstop>
    <tabstop>okPushButton</tabstop>
    <tabstop>cancelPushButton</tabstop>
    <tabstop>fromComboBox</tabstop>
  </tabstops>
  <includes>
    <include location="local">qstring.h</include>
    <include location="local">sockets/url.h</include>
    <include location="local">ui_getaddressform.h</include>
    <include location="local">user.h</include>
    <include location="local">phone.h</include>
  </includes>
  <connections>
    <connection>
      <sender>cancelPushButton</sender>
      <signal>clicked()</signal>
      <receiver>InviteForm</receiver>
      <slot>reject()</slot>
    </connection>
    <connection>
      <sender>okPushButton</sender>
      <signal>clicked()</signal>
      <receiver>InviteForm</receiver>
      <slot>validate()</slot>
    </connection>
    <connection>
      <sender>addressToolButton</sender>
      <signal>clicked()</signal>
      <receiver>InviteForm</receiver>
      <slot>showAddressBook()</slot>
    </connection>
    <connection>
      <sender>hideUserCheckBox</sender>
      <signal>clicked()</signal>
      <receiver>InviteForm</receiver>
      <slot>warnHideUser()</slot>
    </connection>
  </connections>
</ui>
